datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -658,13 +667,13 @@ def process_metric(self, metric, scraper_config, metric_transformers=None):
 
     def poll(self, scraper_config, headers=None):
         """
-        Custom headers can be added to the default headers.
-
-        Returns a valid requests.Response, raise requests.HTTPError if the status code of the requests.Response
-        isn't valid - see response.raise_for_status()
+        Returns a valid requests.Response, otherwise raise requests.HTTPError if the status code of the
+        requests.Response isn't valid - see `response.raise_for_status()`
```

Comment:
```suggestion
        Returns a valid `requests.Response`, otherwise raise requests.HTTPError if the status code of the
        response isn't valid - see `response.raise_for_status()`
```